packages/create-catalyst/src/commands/integration.ts

```diff
@@ -0,0 +1,118 @@
+import { Command } from '@commander-js/extra-typings';
+import { exec as execCb } from 'child_process';
+import { outputFileSync, writeJsonSync } from 'fs-extra/esm';
+import * as semver from 'semver';
+import { promisify } from 'util';
+import * as z from 'zod';
+
+const exec = promisify(execCb);
+
+interface Manifest {
+  name: string;
+  dependencies: string[];
+  devDependencies: string[];
+  environmentVariables: string[];
+}
+
+export const integration = new Command('integration')
+  .requiredOption('--integration-name <name>', 'Formatted name of the integration')
+  .requiredOption('--source-branch <branch>', 'The branch containing your integration source code')
```

Comment:
Can this just be the output of `git rev-parse --abbrev-ref HEAD` (current branch name)? A `--commit-hash` option could override this.